recipes/xmlsec/all/conanfile.py

```diff
@@ -54,9 +54,9 @@ def configure(self):
 
     def build_requirements(self):
         if not self._is_msvc and tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
```

Comment:
I think there's an update to this line 🤔 @madebr or @SpaceIm if you have a second to check

Reply:
yes:

```python
    @property
    def _settings_build(self):
        getattr(self, "settings_build", self.settings)

    def build_requirements(self):
        if not self._is_msvc and self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
            self.build_requires("msys2/cci.latest")
```

Reply:
https://github.com/conan-io/conan-center-index/pull/6853